articles/iot-edge/production-checklist.md

```diff
@@ -316,6 +316,20 @@ You can do so in the **createOptions** of each module. For example:
 
 For the most efficient IoT Edge deployment scenario, consider integrating your production deployment into your testing and CI/CD pipelines. Azure IoT Edge supports multiple CI/CD platforms, including Azure DevOps. For more information, see [Continuous integration and continuous deployment to Azure IoT Edge](how-to-continuous-integration-continuous-deployment.md).
 
+
+### Each IoT Edge device should have its own unique device connection string 
+
+Please make sure unique device connection string is **not** inadvertently **shared** between 2 or more separate IoT Edge devices.
+IoT Edges devices sharing the device connection string may seem to work properly but it can lead to following issues before condition is detected and fixed
+
+- 1.)	Huge spike in module and device connects/disconnects
+- 2.)	Huge spike in module D2C Get Twin operations
+
+Consequence of this may result in unexpected **IoT Hub message quota exhaustion** and increase in **data usage bill** specifically if using cellular or satellite data connection. To check for this condition you can monitor the frequency of connect/disconnects of modules and devices using Azure Diagnostics logs for IoT Hub.
+
+
+
```

Comment:
Do you want to add anything about customers that use certificate-based authentication? Do they have the same issues if they use an identity certificate on two separate IoT Edge devices, or no?

Reply:
Made changes